### ISSUE TITLE
Say contents of URL does nothing

### ISSUE THREAD

**Issue by ToonTalk** [jmoenig/Snap--Build-Your-Own-Blocks#1946]
The following does nothing

![image](https://user-images.githubusercontent.com/2138696/33277288-659fa640-d398-11e7-91bf-576bcba3b187.png)

While just the URL reporter block works fine.

**Comment by YPIMG** [comment]
![screenshot 2017-11-27 at 10 47 43](https://user-images.githubusercontent.com/32828248/33278214-74b3ea70-d360-11e7-8e37-8b5fe1863c84.png)
Seems to work for me.

**Comment by jguille2** [comment]
For me it is also working ok!

**Comment by ToonTalk** [comment]
I tried it several times in Chrome on Windows 10. Nothing happens.

Just now I tried it in Edge and it says:

![image](https://user-images.githubusercontent.com/2138696/33279189-fbfadf88-d39d-11e7-84e1-1553a0c337a1.png)

But there is sort of a "say":

![image](https://user-images.githubusercontent.com/2138696/33279199-0a770104-d39e-11e7-89a6-4a050960dc39.png)

In Firefox 57.0 I repeatedly get

![image](https://user-images.githubusercontent.com/2138696/33279223-20bdac10-d39e-11e7-9517-ce8166cbde15.png)

**Comment by brianharvey** [comment]
This sounds like a limit on the length of the input to SAY maybe?

**Comment by ToonTalk** [comment]
Why does SAY work so different from clicking on a reporter? The URL brick shows the first few hundred characters and then "..."

**Comment by brianharvey** [comment]
Oh, it's a bug, I was just suggesting what the bug might be.

**Comment by ToonTalk** [comment]
Right. I was just suggesting that maybe the code for showing the value when a reporter is clicked should also be used for SAY since it seems more robust. Aren't they conceptually almost the same thing?

**Comment by DarDoro** [comment]
Seems to work for me on Windows 10 Pro (64)
Chrome  61.0.3163.100 (64)
Firefox Quantum 57.0 (64)
"bar codes" output
IE 11.1176 (64)
Edge (64)

Thera is also problem with URL itself. More info about NS_ERROR_FAILURE can be found in browser console log.

**Comment by ToonTalk** [comment]
I suspect that Brian is right that it depends upon the length of the string.

It works fine with URL snap.berkeley.edu/snapsource/snap.html but not URL snap.berkeley.edu

Could the different results for Chrome depend upon the version? I'm running  62.0.3202.94

**Comment by ToonTalk** [comment]
And Chrome on Android crashes the tab.
